fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -2699,27 +2714,6 @@ ACTOR void setupAndRun(std::string dataFolder,
 	state Optional<TenantName> defaultTenant;
 	state Standalone<VectorRef<TenantNameRef>> tenantsToCreate;
 	state TenantMode tenantMode = TenantMode::DISABLED;
-	// If this is a restarting test, restartInfo.ini is read in restartSimulatedSystem
-	// where we update the defaultTenant and tenantMode in the testConfig
-	// Defer setting tenant mode and default tenant until later
-	if (!rebooting) {
```

Comment:
Don't we still want to make sure this check is done?

Reply:
part of this code block is moved to `setTenantMode` and part is moved to lines below after `setupSimulatedSystem`.